frappe/public/js/frappe/list/list_view_select.js

```diff
@@ -108,7 +108,9 @@ frappe.views.ListViewSelect = class ListViewSelect {
 				action: () => this.set_route("tree"),
 			},
 			Kanban: {
-				condition: this.doctype != "File",
+				condition:
+					this.doctype != "File" &&
+					!(frappe.listview_settings[this.doctype] != null && frappe.listview_settings[this.doctype].skip_kanban),
```

Comment:
This is a hacky solution and only solves your specific use case. 


Add a generic version that's opposite of `add_view_to_menu` like `hide_view_from_menu` and then you can call it hide specific views you dont like.

Reply:
Thanks for reply and hint.
I will be less hacky on next pull requests.